### PR TITLE
add scheduler main.go

### PR DESCRIPTION
**Pull request by allwmh**
### Ⅰ. Describe what this PR does
Create scheduler main function, it support add custom plugins.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,6 +97,10 @@ docker-build-koordlet: ## Build docker image with the koordlet.
 docker-build-koord-manager: ## Build docker image with the koord-manager.
 	docker build --build-arg MODULE=koord-manager -t ${KOORD_MANAGER_IMG} .
 
+.PHONY: docker-build-scheduler
+docker-build-koord-scheduler: ## Build docker image with the scheduler.
```
Comment (eahydra) [inline Makefile:101]: add build-koord-scheduler like as build-koord-manager

File: cmd/koord-scheduler/main.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import (
+	"math/rand"
+	"os"
+	"time"
+
+	"k8s.io/component-base/logs"
+	"k8s.io/kubernetes/cmd/kube-scheduler/app"
+)
+
+//TODO
```
Comment (eahydra) [inline cmd/koord-scheduler/main.go:28]: remove the TODO comment

File: Makefile (hunk 1 above)
Comment (allwmh) [reply Makefile:106]: fix

File: cmd/koord-scheduler/main.go (hunk 2 above)
Comment (allwmh) [reply cmd/koord-scheduler/main.go]: fix

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**